packages/web-monetization-demo/package.json

```diff
@@ -26,20 +26,20 @@
     "lint:all": "yarn lint 'src/**/*.{ts,tsx}' 'test/**/*.{ts,tsx}'",
     "precommit": "echo lint-staged runs from root",
     "prettier": "prettier --write '*.{ts,tsx,js,html,jsx,md}' '{src,test}/**/*.{ts,tsx,js,html,jsx,md}'",
-    "receipt-verifier": "REDIS_URI=mock SPSP_ENDPOINT=http://localhost:4000/spsp/~niq SPSP_PROXY_PORT=4001 VERIFIER_PORT=4002 receipt-verifier",
-    "receiver:dev": "ts-node-dev -r tsconfig-paths/register -P test/tsconfig.json --respawn --transpile-only src/receiver/main.ts",
+    "receiver:dev": "ts-node -r tsconfig-paths/register -P test/tsconfig.json --transpile-only src/receiver/main.ts",
```

Comment:
I was running into multiple issues with `ts-node-dev` (maybe since the switch to yarn 2?)

Reply:
Hrmmmmm, I see
Yes, it could be `yarn 2` related
Potentially this:
https://github.com/coilhq/web-monetization-projects/issues/1271#issue-737612554

Reply:
For the record:

1. Problems parsing `//` comments in tsconfig.json which TS itself allows
   1. Just remove comments
2. Problems solved by using `./src/module.ts` rather than `src/module.ts`
   1. Update package.json
3. Problems with tsconfig resolution, such that `TypeError: tsconfig_1.resolveSync is not a function`
   1. ???

Reply:
Yarn 2, using zips for 3rd party dependencies, is quite annoying when you want to edit files and add debug logging. 
I'm yet to learn a good workflow for this
Hopefully there's a yarn 2 "trick" for putting a dependency into some easily editable form/location

Reply:
In an effort to add some debug logs to `ts-node-dev` I set `nodeLinker: node_modules` in the `.yarnrc.yml` file. This would allow me to edit the sources, as pnp links to zip files directly. This (after steps 1&2 above) itself allowed ts-node-dev to run. I also tried updating to the latest release version of yarn, as well as the latest development version "from sources" but all to no avail.

Note that you can run `yarn node` from the demo package folder, then require('tsconfig') and strangely you will get the root tsconfig.json:
![image](https://user-images.githubusercontent.com/525211/102167063-30e6b100-3ec0-11eb-8320-905cf81ce715.png)

If ts-node-dev is looking to require a `tsconfig` module, expecting it to have resolveSync function, but instead gets the tsconfig.json, that would explain the behaviour seen.

Clearly there is *something* wrong with the pnp linking

Reply:
> Yarn 2, using zips for 3rd party dependencies, is quite annoying when you want to edit files and add debug logging.

Vim can edit files inside a zip file without extracting

Reply:
I knew some Vim fanb0i would come and mock us clunky Java IDE users :/ 

:)